tests/test_server_basic.py

```diff
@@ -41,16 +41,16 @@
                     query_type="Real",
                     instrument="isgri",
                     product_type="isgri_image",
-                    osa_version="OSA10.2",
-                    E1_keV=20.,
-                    E2_keV=40.,
+                    # osa_version="OSA10.2",
+                    # E1_keV=20.,
+                    # E2_keV=40.,
```

Comment:
Will you uncomment those back? I am not sure this won't brake any test...

Reply:
These are integral-specific parameters, which are not really used (they are not defined in the empty* instruments, which are used in every test of this file). That's why I commented them, otherwise, many tests will break with this new proposed behavior of filtering unknown arguments.
Like this, all tests pass.

Reply:
which tests dont pass?

Reply:
Those where job_id is being compared with the one, calculated in test using make_hash on params

Reply:
Then those tests should be modified accordingly: by removing those entries (eg E1, E2 etc) from the dict used to calculate the `job_id`for the comparison